app/renderer/components/preferences/payment/ledgerTable.js

```diff
@@ -260,12 +265,20 @@ class LedgerTable extends ImmutableComponent {
             onClick={this.togglePinSite.bind(this, this.getHostPattern(synopsis), publisherKey, !pinned, percentage)}
             data-test-pinned={pinned}
           />
-          <span className={css(
+          {
+            !this.props.inProgress
```

Comment:
Can we instead of duplicating span and copy pasting styles, have just one function for this action and then inside that function we can trigger different stuff based on the flag. This way it's easier to refactor things or add some new classes to the span, less error prone.

Reply:
Done.